packages/sdk/contractkit/src/wrappers/FeeHandler.ts

```diff
@@ -0,0 +1,76 @@
+import { Address } from '@celo/connect'
```

Comment:
would you add a changeset for this like 

`yarn cs add` 

@celo/contractkit  => minor
"add FeeHandler Wrapper"

Reply:
will address this after I get the CI green